_episodes/12-cluster.md

```diff
@@ -101,7 +141,24 @@ supplied by the instructors. You may be asked for your password. Watch out: the
 characters you type after the password prompt are not displayed on the screen.
 Normal output will resume once you press `Enter`.
 
-## Where Are We?
+You may have noticed that the prompt changed when you logged into the remote
+system using the terminal (if you logged in using PuTTY this will not apply
+because it does not offer a local terminal). This change is important because
+it makes it clear on which system the commands you type will be run when you
```

Comment:
```suggestion
it can help you distinguish on which system the commands you type will be run when you
```

because clarity is in the eye of the beholder